### DISCUSSION TITLE
thirdly_btn

### DISCUSSION THREAD

**Discussion by mprechtel** [orestbida/cookieconsent#540] [Q&A]
Hi,

it is possible to display a third button (thirdly_btn) at the initial call (modal) that rejects all cookies (reject_all_btn).
Thank you

**Comment by mprechtel** [comment]
Like this
![grafik](https://github.com/orestbida/cookieconsent/assets/4092708/3b392ec3-9b5a-4d46-bac0-8959238d388d)

**Comment by orestbida** [comment]
Hi @mprechtel,

this is possible in [v3](https://cookieconsent.orestbida.com/) only. [Playground](https://playground.cookieconsent.orestbida.com/).

**Comment by mprechtel** [comment]
Hi, thanks for the info. Great tool.
